[WARNING] ME(368786:281473456531760,MainProcess):2025-08-28-16:19:07.750.252 [mindspore/run_check/_check_version.py:412] Can not find the tbe operator implementation(need by mindspore-ascend). Please check whether the Environment Variable PYTHONPATH is set. For details, refer to the installation guidelines: https://www.mindspore.cn/install
/home/ma-user/anaconda3/envs/MindSpore/lib/python3.9/site-packages/numpy/core/getlimits.py:500: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/home/ma-user/anaconda3/envs/MindSpore/lib/python3.9/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
/home/ma-user/anaconda3/envs/MindSpore/lib/python3.9/site-packages/numpy/core/getlimits.py:500: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/home/ma-user/anaconda3/envs/MindSpore/lib/python3.9/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
[WARNING] ME(368786:281473456531760,MainProcess):2025-08-28-16:19:12.152.851 [mindspore/run_check/_check_version.py:412] Can not find the tbe operator implementation(need by mindspore-ascend). Please check whether the Environment Variable PYTHONPATH is set. For details, refer to the installation guidelines: https://www.mindspore.cn/install
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:12.154.657 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:485] Connect] Connection 17 source: 127.0.0.1:38532, destination: 127.0.0.1:8118
[WARNING] DISTRIBUTED(368786,ffff2ded91e0,python):2025-08-28-16:19:12.154.657 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:79] ConnectedEventHandler] Connection from 127.0.0.1:38532 to 127.0.0.1:8118 is successfully created. System errno: Success
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:12.154.708 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:494] Connect] Waiting for the state of the connection to 127.0.0.1:8118 to be connected...Retry number: 1
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:12.654.896 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:485] Connect] Connection 18 source: 127.0.0.1:38546, destination: 127.0.0.1:8118
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:12.654.932 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:494] Connect] Waiting for the state of the connection to 127.0.0.1:8118 to be connected...Retry number: 2
[WARNING] DISTRIBUTED(368786,ffff2eedb1e0,python):2025-08-28-16:19:12.654.931 [mindspore/ccsrc/distributed/rpc/tcp/tcp_comm.cc:79] ConnectedEventHandler] Connection from 127.0.0.1:38546 to 127.0.0.1:8118 is successfully created. System errno: Success
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:13.155.369 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:246] BuildCluster] Topology build timed out., retry(1/1200).
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:13.655.500 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:246] BuildCluster] Topology build timed out., retry(2/1200).
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:14.155.664 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:246] BuildCluster] Topology build timed out., retry(3/1200).
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:14.655.856 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:249] BuildCluster] Cluster is successfully initialized.
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:14.656.161 [mindspore/ccsrc/distributed/cluster/cluster_context.cc:356] PostProcess] This node 5 rank id: 5
[WARNING] DISTRIBUTED(368786,ffffa563e930,python):2025-08-28-16:19:20.921.013 [mindspore/ccsrc/distributed/collective/collective_manager.cc:338] CreateCommunicationGroup] Start to create communication group: hccl_world_group [const vector]{0, 1, 2, 3, 4, 5, 6, 7}, async: 0, submit_now: 1
[WARNING] DISTRIBUTED(368786,fffea97fa1e0,python):2025-08-28-16:19:20.922.198 [mindspore/ccsrc/distributed/collective/collective_manager.cc:962] CreateDeviceCommunicator] Begin initialize communication group on the device side: hccl_world_group
[WARNING] DEVICE(368786,fffea8ff91e0,python):2025-08-28-16:19:20.922.445 [mindspore/ccsrc/plugin/res_manager/ascend/collective/ascend_communication_group.cc:165] InitializeByRankTable] Start to initialize communicator by HcclCommInitClusterInfoConfig for hccl_world_group, hcclBufferSize is 200 MB. hcclDeterministic is 0
[WARNING] DEVICE(368786,fffea8ff91e0,python):2025-08-28-16:19:21.005.433 [mindspore/ccsrc/plugin/res_manager/ascend/collective/ascend_communication_group.cc:175] InitializeByRankTable] End to initialize communicator by HcclCommInitClusterInfoConfig for hccl_world_group
[WARNING] DISTRIBUTED(368786,fffea97fa1e0,python):2025-08-28-16:19:21.006.230 [mindspore/ccsrc/distributed/collective/collective_manager.cc:973] CreateDeviceCommunicator] End initialize communication group on the device side: hccl_world_group
[WARNING] ME(368786:281473456531760,MainProcess):2025-08-28-16:19:34.559.853 [mindspore/train/serialization.py:1860] For 'load_param_into_net', remove parameter prefix name: ema., continue to load.
Scanning 'coco/val2017.cache.npy' images and labels... 5000 found, 0 missing, 48 empty, 0 corrupted: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████| 5000/5000 [00:00<?, ?it/s]Scanning 'coco/val2017.cache.npy' images and labels... 5000 found, 0 missing, 48 empty, 0 corrupted: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████| 5000/5000 [00:00<?, ?it/s]
................